Bottom: 3313c96dcc22cbf1f134c49c35a16540fde9fc11
Top:    a179935c6eb521104f26ec12dee4c55f0fd2745d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:10:14 +0100

Refresh of hid-gtk-enable-transparent-lin

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 0b76c71..5bc13b6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
